ui/src/components/Hotflip.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+
+import { RedToken, GreenToken, TransparentToken } from './Tokens';
+
+// NOTE: The original HotFlipComponent handled targeted words to flip.  This was used in the
```

Comment:
```suggestion
// NOTE: The original HotFlipComponent handled targeted words to flip.  This was only used in the
```

I think that's the case, let me know if it's not.